### ISSUE TITLE
Receiving donations without payment processors

### ISSUE THREAD

**Issue by filips123** [liberapay/liberapay.com#1385]
I want to receive donations, but I currently can't set up payment processors.
Can I still accept donations, but only receive them when I will set payment processors? I want do do some king of holding that money directly in Liberapay.

**Comment by Changaco** [comment]
No, Liberapay can't hold money for you. Donors can pledge to donate to you but they won't be able to actually send the money until you've configured payment processing.

**Comment by filips123** [comment]
So if donors make donation to me, it won't be transferred, but when I set up processors, it will be automatically transferred?

**Comment by Changaco** [comment]
If it happens in the near future, no, because automatic transfers aren't implemented yet (#483). In fact we don't even send notifications to donors in the case you're describing.
